course_contents/1_intro/notes/06. Identity Operators.md

```diff
@@ -0,0 +1,37 @@
+## IS
+- Return True if the operands have the same identity
+
+Example:
+```
+users = ['James', 'Charlie', 'Ana']
+print(id(users))  # 2425142160952
+
+users_copy = users
+print(id(users_copy))  # 2425142160952
+
+print(users_copy is users)  # True
+
+users_copy = ['James', 'Charlie', 'Ana']
+print(id(users_copy))  # 2425142558551
+
+print(users_copy is users)  # False
+```
+
+## IS NOT
+- Returns True if the operands don't have the same identity
+
+Example:
+```
+users = ['James', 'Charlie', 'Ana']
+print(id(users))  # 2425142160952
+
+users_copy = users
+print(id(users_copy))  # 2425142160952
+
+print(users_copy is users)  # False
+
+users_copy = ['James', 'Charlie', 'Ana']
+print(id(users_copy))  # 2425142558551
+
+print(users_copy is users)  # True
+```
```

Comment:
Not terribly important, but missing the syntax highlighting here.

Reply:
Fixed